scenarios/kubernetes_e2e.py

```diff
@@ -92,17 +92,22 @@ def main(args):
         for env in args.env_file:
             cmd.extend(['--env-file', test_infra(env)])
 
-    gce_ssh = '/workspace/.ssh/google_compute_engine'
-    gce_pub = '%s.pub' % gce_ssh
-    service = '/service-account.json'
+    if args.gce_ssh:
+        gce_ssh = '/workspace/.ssh/google_compute_engine'
+        gce_pub = '%s.pub' % gce_ssh
+        cmd.extend([
+          '-v', '%s:%s:ro' % (args.gce_ssh, gce_ssh),
+          '-v', '%s:%s:ro' % (args.gce_pub, gce_pub),
+          '-e', 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE=%s' % gce_ssh,
+          '-e', 'JENKINS_GCE_SSH_PUBLIC_KEY_FILE=%s' % gce_pub])
+
+    if args.service_account:
+        service = '/service-account.json'
+        cmd.extend([
+            '-v', '%s:%s:ro' % (args.service_account, service),
+            '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
```

Comment:
So I'm trying to figure out how to run kops on AWS tests with our new JSON driven config; so far I have this:

`JENKINS_UPLOAD_TO_GCS=n scenarios/kubernetes_e2e.py --env-file=platforms/kops-aws.env --env-file=jobs/ci-kubernetes-e2e-kops-aws.env --entrypoint=/workspace/kops-e2e-runner.sh --tag=v20170209-9b5f1817-dirty `

I don't have a service-account.json configured at all (nor a GCE SSH key) - and I don't particularly want to install one just in case I start uploading build results :-)

AFAICT, bootstrap calls the same command?